docs/app-store-connect-api.md

```diff
@@ -36,7 +36,7 @@ Below are the statuses of each tool:
 
 1. Create a new App Store Connect API Key in the [Users page](https://appstoreconnect.apple.com/access/api)
   - For more info, go to the [App Store Connect API Docs](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api)
-  - Select the "Keys" tab
+  - Open the ‘Integrations’ tab, select ‘App Store Connect API,’ and create a new TEAM API using the ‘+’ button. 
```

Comment:
```suggestion
  - Open the `Integrations` tab, select `App Store Connect API`, and create a new "Team Key" using the `+` button
    - Note that if you can't see the `App Store Connect API` tab, this means you don't have permission yet. Please refer to the docs above to know how to get this permission
```

Reply:
Normally I prefer ending every sentence with a period, but the pattern here is to not have an ending period so I opted for consistency 🤕